templates/snippets/shared/icons/star.liquid

```diff
@@ -0,0 +1,9 @@
+{%- default style_class: blank -%}
+{%- default width: blank -%}
+
+<svg class="icon-star testimonial__star-rating" aria-hidden="true" focusable="false" role="presentation"
```

Comment:
`testimonial__star-rating` contains an extra underscore.

These classes should be passed from calling the snippet in the previous component. This way we make it reusable.